### DISCUSSION TITLE
Github Project view is strange on the mobile app.

### DISCUSSION THREAD

**Discussion by major-sanyi** [community/community#76725] [Mobile]
If I want to select 1 project and work on my IPad, I can hide the:

- left side navbar
- the issues list, to focus on 1 issue. 

If I want to hide the page in—between which is listing all of my projects, I can't, which is strange since I might jump between tasks, but jump through projects less. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.139.0 (115734377)
Device: iPad14,3 (iOS 17.1.1)
</details>

**Comment by taki-on** [comment]
Thank you for reaching out to us @major-sanyi. We are aware of this issue and a fix is on the way 😃
